src/bin/list_store.rs

```diff
@@ -0,0 +1,311 @@
+// "List Store" gtk3 demo ported to gtk-rs by Ömer Sinan Ağacan <omeragacan@gmail.com>
+
+extern crate gio;
+extern crate gtk;
+
+use gio::prelude::*;
+use gtk::prelude::*;
+
+use std::env::args;
+use std::rc::Rc;
+
+fn main() {
+    let application =
+        gtk::Application::new("com.github.gtk-rs.examples.list-store", Default::default())
+            .expect("Initialization failed...");
+
+    application.connect_startup(|app| {
+        build_ui(app);
+    });
+
+    application.connect_activate(|_| {});
+
+    application.run(&args().collect::<Vec<_>>());
+}
+
+#[repr(i32)]
+enum Columns {
+    Fixed = 0,
+    Number,
+    Severity,
+    Description,
+    Pulse,
+    Icon,
+    Active,
+    Sensitive,
+}
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(application);
+
+    window.set_title("List Store");
+    window.set_border_width(10);
+    window.set_position(gtk::WindowPosition::Center);
+    window.set_default_size(280, 250);
+
+    let vbox = gtk::Box::new(gtk::Orientation::Vertical, 8);
+    window.add(&vbox);
+
+    let label = gtk::Label::new(
+        "This is the bug list (note: not based on real data, it would be \
+         nice to have a nice ODBC interface to bugzilla or so, though).",
+    );
+    vbox.add(&label);
+
+    let sw = gtk::ScrolledWindow::new(None::<&gtk::Adjustment>, None::<&gtk::Adjustment>);
+    sw.set_shadow_type(gtk::ShadowType::EtchedIn);
+    sw.set_policy(gtk::PolicyType::Never, gtk::PolicyType::Automatic);
+    vbox.add(&sw);
+
+    let model = Rc::new(create_model());
+    let treeview = gtk::TreeView::new_with_model(&*model);
+    treeview.set_vexpand(true);
+    treeview.set_search_column(Columns::Description as i32);
+
+    sw.add(&treeview);
+
+    add_columns(model.clone(), &treeview);
+
+    window.show_all();
+
+    let model = model.clone();
+    timeout_add(80, move || spinner_timeout(&model));
```

Comment:
Hm how does this compile? You don't import that function directly anymore via `gtk::*`, so it probably would have to be prefixed with `gtk::` now?